File: atcoder/abc236/abc236_a.java
package atcoder;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    //problem abc236_a
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader());
        char[] s = br.readLine().toCharArray();
        String[] nums = br.readLine().split(" ");
        int a = Integer.parseInt(nums[0]);
        int b = Integer.parseInt(nums[1]);
        char temp = s[a-1];

        s[a-1] = s[b-1];
        s[b-1] = temp;
        System.out.println(s.toString());
    }
}

File: atcoder/abc236/abc236_a.kt
package atcoder

import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val In = BufferedReader(InputStreamReader(System.`in`))
    var s = In.readLine()!!.toMutableList()
    val nums = In.readLine()!!.split(" ").map { it.toInt() }
    val temp = s[nums[0]-1]
    s[nums[0]-1] = s[nums[1]-1]
    s[nums[1]-1] = temp
    println(s)
}

